src/components/passwordreset/feedbackContainer/index.js

```diff
@@ -0,0 +1,32 @@
+import { connect } from 'react-redux';
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Feed from './feedback';
+// import Footer from '../footer';
```

Comment:
I think it would be better if you remove these commented lines if you are using them.

Reply:
Removed